README.md

```diff
@@ -5,33 +5,21 @@
 experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental) 
 <!-- badges: end -->
 
-The goal of `workflow.scenario.preparation` is to prepare all input scenario datasets required as input by both the [pactaverse](https://rmi-pacta.github.io/pactaverse/) R packages (for investors), and [r2dii](https://rmi-pacta.github.io/r2dii.analysis/) R packages (for banks).
-
-## Create a `.env` file
-
-Running this workflow requires a file called `.env` in the root directory, that looks like...
-
-```
-HOST_INPUTS_PATH=/users/scenario_data/inputs
-HOST_OUTPUTS_PATH=/users/scenario_data/outputs
-R_CONFIG_ACTIVE=YYYYQQ
-```
-
-This file will specify the location of the input and output directories, and the active configuration set-up (see `config.yml` for more information).
+Welcome to `workflow.scenario.preparation`! This tool is designed to streamline the preparation of input scenario datasets for use in either [workflow.data.preparation](https://github.com/RMI-PACTA/workflow.data.preparation) or the [r2dii](https://rmi-pacta.github.io/r2dii.analysis/) R packages.
 
 ## Required Input Files
 
-All required files must exist at `$HOST_INPUTS_PATH`.
+Ensure the following files exist in your input directory:
 
 ### General files
 
-These files are required for the preparation of all scenarios:
+These files are essential for preparing all scenarios:
 
 - `technology_bridge.csv`
```

Comment:
Flag an issue to document format for this file.

Reply:
See https://github.com/RMI-PACTA/workflow.scenario.preparation/issues/12